### ISSUE TITLE
problems when loaded with other packages

### ISSUE THREAD

**Issue by pedritomelenas** [martin-leuner/alcove#9]
If you load FinIng and then alcove, you get an error 
```
Error, operation `Hyperplanes' has been declared as a one argument Operation and cannot also be an Attribute at /Users/pedro/lib/gap-4.9.0/lib/oper.g:1238 called from
<function "DeclareAttribute">( <arguments> )
 called from read-eval loop at /Users/pedro/lib/gap-4.9.0/pkg/alcove/gap/Matro\
id.gd:338
```
Also when you do LoadAllPackages() you get something like 
```
Error, operation `IsConnected' was not created as a property, use `DeclareOperation' at /Users/pedro/lib/gap-4.9.0/lib/oper.g:1502 called from
<function "DeclareProperty">( <arguments> )
 called from read-eval loop at /Users/pedro/lib/gap-4.9.0/pkg/alcove/gap/Matro\
id.gd:506
you can 'quit;' to quit to outer loop, or
you can 'return;' to continue
```

Thank you!
Pedro

**Comment by martin-leuner** [comment]
Hi Pedro,
sorry for taking so long to get back to you, and thanks for the report!
The conflicts should be fixed in commit 4abaded6c04b3b6005a1ed46bfb5c138f4f3fab9.
Cheers,
Martin

**Comment by pedritomelenas** [comment]
Thanks!
